### ISSUE TITLE
Winetricks look & feel nightmare

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#94]
```
What steps will reproduce the problem?
1.one of my app complaint about msvcp80.dll not implemented, even if wine 
1.3.22 have it builtin, and adding it as native via wineconfig dont help.
2.wanted to install msvcp80.dll using winetricks: latest version from wine-team 
ppa on ubuntu natty i386
3.actual winetricks with its multiple check boxes are simply adding confusion 
and worst have removed the usefull details that was provided by older 
winetricks version.

What is the expected output? What do you see instead?
As we can only choose vc2005 (guess that msvcp80 is inside but who knows now ?) 
whithout knowing what it brings (ok we can gloogled around it, but removing 
details are very bad idea), please bring back the "old" fashion.

What version of the product are you using? On what operating system?
see details above.

Please provide any additional information below.


```

Original issue reported on code.google.com by `sparb...@numericable.fr` on 19 Jun 2011 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
The details are fine here on Ubuntu 11.04.

What operating system and desktop are you using?

Can you provide a screenshot of the screen with the useful details missing?
```

Original comment by `daniel.r...@gmail.com` on 20 Jun 2011 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
No response, likely invalid.
```

Original comment by `austinenglish@gmail.com` on 16 Jun 2013 at 12:42
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Why did I ask him what OS he was using?  Man, sometimes I just don't read.

BTW, the function was
Unimplemented function 
msvcp80.dll.??$?9DU?$char_traits@D@std@@V?$allocator@D@1@@std@@YA_NABV?$basic_st
ring@DU?$char_traits@D@std@@V?$allocator@D@2@@0@PBD@Z 
which was implemented in June 2011 by 9e9d3cc90293940ef856b233cfb14d8e2c33f9f3

The description still says 
"Visual C++ 2005 libraries (mfc80,msvcp80,msvcr80)", so the info he wants is 
still there.
It's possible he didn't have zenity installed, or something.

Oh, well.
```

Original comment by `daniel.r...@gmail.com` on 16 Jun 2013 at 3:04
